components/card/textOnly/preview.html

```diff
@@ -1,18 +1,11 @@
-<h4 class="title fs-l">Text Only Card Button</h4>
+<h4 class="title fs-l">Text Only Card </h4>
 <div class="btn-wrapper preview-container rounded-s p-mid flex flex-center fullwidth">  
   <section class="card pos-rel">
-    <!-- <div class="flex flex-col pos-rel text-over-media">
-      <img class="card-img" src="https://placeimg.com/200/100" alt="card image">
-      
-    </div> -->
```

Comment:
remove extra comments

Reply:
It was already resolved in the latest [commit](https://github.com/tushartiwari7/elevate-ui/pull/13/commits/67ee2f8af1be75c7456cfdfa1c8b9b34c8b22588). please have a look again @Bhakti199